system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.hpp

```diff
@@ -34,18 +34,17 @@ class LinuxCpuMeasurementNode : public system_metrics_collector::PeriodicMeasure
 {
 public:
   /**
-   * Construct a LinuxCpuMeasurementNode
+   * Construct a LinuxCpuMeasurementNode.
+   * The following parameters may be set via the rclcpp::NodeOptions:
+   * `measurement_period`: the period of this node, used to read measurements
+   * `publish_period`: the period at which metrics are published. 0 ms means don't publish
```

Comment:
Same comment per the 0ms docstring.

Reply:
This is the same comment as https://github.com/ros-tooling/system_metrics_collector/pull/73#discussion_r367700566.